argus/recipes/cloud/windows.py

```diff
@@ -324,6 +323,9 @@ def prepare_cbinit_config(self, service_type):
         super(CloudbaseinitMockServiceRecipe, self).prepare_cbinit_config(service_type)
         LOG.info("Inject guest IP for mocked service access.")
 
+
+        # TODO(mmicu): The service type is specific to each scenario,
+        #              find a way to pass it.
```

Comment:
for the todo indentation, why don't we keep it clean and in sync with the openstack one?
https://github.com/openstack/nova/blob/bc5035343d366a18cae587f92ecb4e871aba974a/nova/scheduler/filters/retry_filter.py#L37